ui/app/AppLayouts/Onboarding/shared/CreatePasswordModal.qml

```diff
@@ -171,15 +173,15 @@ ModalPopup {
                         passwordValidationError = qsTr("Incorrect password")
                     }
                     else {
-                        //Global.applicationWindow.prepareForStoring(repeatPasswordField.text, true)
```

Comment:
Should we not remove these entirely then?

Reply:
@PascalPrecht yes, in case if this is not in use any more somewhere during the onboarding process? I saw that complete else block was left when onboarding process was updated and just uncomment the line so it does what it did before.